test/stress/large_microbench.c

```diff
@@ -0,0 +1,35 @@
+#include "test/jemalloc_test.h"
+#include "test/bench.h"
+
+static void
+large_mallocx_free(void) {
+	void *p = mallocx(128 * 1024, 0);
+	if (p == NULL) {
+		test_fail("Unexpected mallocx() failure");
+		return;
+	}
```

Comment:
nit: `assert_ptr_not_null()` to be more compact?

Reply:
Will do.